frontend/src/components/ProjectList/components/Dataset/components/DownloadDataset/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from "react";
+import Modal from "src/components/common/Modal";
+import Content from "./components/Content";
+
+const DownloadDataset: FC = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const toggleOpen = () => setIsOpen(!isOpen);
+
+  return (
+    <div>
+      <button onClick={toggleOpen}>Download</button>
+      <Modal isOpen={isOpen} onClose={toggleOpen}>
+        <Content />
```

Comment:
The [part 2 ticket](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/corpora-data-portal/518) will mostly happen within this component, so hopefully this will minimize the merge conflict!